scripts/release/build.ts

```diff
@@ -52,18 +54,63 @@ export const applyMirroring = (feature: WalkOutput): void => {
   }
 };
 
+const getPreviousVersion = (
+  browser: BrowserName,
+  version: VersionValue,
+): VersionValue => {
+  if (typeof version === 'string' && !version.startsWith('≤')) {
+    const browserVersions = Object.keys(bcd.browsers[browser].releases).sort(
+      compareVersions,
+    );
+    const currentVersionIndex = browserVersions.indexOf(version);
+    if (currentVersionIndex > 0) {
+      return browserVersions[currentVersionIndex - 1];
+    }
+  }
+
+  return version;
+};
+
+/**
+ * Add version_last
+ * @param {WalkOutput} feature The BCD to transform
+ * @returns {void}
+ */
+export const addVersionLast = (feature: WalkOutput): void => {
+  for (const [browser, supportData] of Object.entries(
+    feature.compat.support as InternalSupportStatement,
+  )) {
+    if (Array.isArray(supportData)) {
+      for (let i = 0; i < supportData.length; i++) {
+        if (supportData[i].version_removed) {
+          (feature.data as any).__compat.support[browser][i].version_last =
+            getPreviousVersion(
+              browser as BrowserName,
+              supportData[i].version_removed,
+            );
+        }
```

Comment:
Can we avoid the duplication between lines 85-91 and 94-98? Also, shouldn't we check for `version_removed` as well below?

Reply:
They're not exactly duplicated -- one is remapping an array, the other just a single object.  I've made some updates to the code to better reflect this!